README.md

```diff
@@ -3,7 +3,7 @@
 **Tags:** Pantheon, hosting  
 **Requires at least:** 4.9  
 **Tested up to:** 6.1  
-**Stable tag:** 0.4.1  
+**Stable tag:** 0.5.0  
```

Comment:
There's no actual changes to the functionality of the plugin, all of this stuff is largely cleanup and dependency bumps. I think we can just call this `0.4.2`.